target/linux/ramips/dts/VOCORE2L.dts

```diff
@@ -0,0 +1,52 @@
+/dts-v1/;
+
+#include "VOCORE2.dtsi"
+
+/ {
+	model = "VoCore2-Lite";
+
+	memory@0 {
```

Comment:
One only slightly related question: Is the correct amount of memory (auto)detected without a memory node?